File: Managemate/ManagemateAPI/ManagemateAPI/Management/M_Item_On_Receipt/Input_Objects/Edit_Item_On_Receipt_Data.cs
using ManagemateAPI.Management.M_Session.Input_Objects;

namespace ManagemateAPI.Management.M_Item_On_Receipt.Input_Objects
{
    public class Edit_Item_On_Receipt_Data
    {
        public Session_Data session { get; set; }

        public long id { get; set; }

        public long item_id_FK { get; set; }

        public double count { get; set; }

        public double summary_weight { get; set; }

        public string annotation { get; set; }

    }
}

File: Managemate/ManagemateAngular/src/app/shared/interfaces/API_Input_Models/Item_On_Receipt_Models/Edit_Item_On_Receipt_Data.ts
import {Session_Data} from "../../API_Other_Models/Session_Models/SessionData";

export interface Edit_Item_On_Receipt_Data{
    session: Session_Data;
    
    id:number;
    
    receipt_id_FK : number;
    item_id_FK : number;
    count : number;
    summary_weight : number;
    annotation : string;
}